### ISSUE TITLE
Screen sharing does not work

### ISSUE THREAD

**Issue by chriskalos** [17hoehbr/Discord-M1#2]
Even with the screen recording permission turned on for the app, the screen sharing dialog doesn't come up when I press the screen share button, which is a huge issue for me as I share my screen on Discord almost every day.

**Comment by bryce-hoehn** [comment]
Yeah I've been trying to get this working but so far nothing has worked. I think it has to do with how electron handles permissions. When I try to screen share in Chrome I get a dialog asking if I would like to allow Discord to record my screen but electron just doesn't do anything.

**Comment by spielers** [comment]
yeap same with me electron needs permission from mac to share the screen same as chrome. but due to its versatile nature, it doesn't ask

**Comment by bryce-hoehn** [comment]
I have also tried manually whitelisting Discord in the Screen Recording section in Security & Privacy but that does not seem to do anything either, although it will probably be necessary for the eventual solution as well.

**Comment by PaddiM8** [comment]
I think I saw some issue about this on the Nativefier or Electron repo, saying that Electron doesn't have its own dialog for this. Might be difficult to get this to work..

**Comment by bryce-hoehn** [comment]
Yeah unfortunantely it seems that way. What I don't quite understand is that in the web version I get permission dialogs whenever I try to share my webcam or my screen, whereas in Electron I don't get prompts for either. Despite this though the webcam seems to share just fine.

**Comment by Casuallynoted** [comment]
Yeah, this is really strange. Hopefully this getting solved on the Electron side would fix it.
